pkg/cmd/deploy.go

```diff
@@ -21,10 +21,9 @@ import (
 	"io"
 	"os/exec"
 
+	"github.com/ghodss/yaml"
```

Comment:
or it can be stdlib, 3rd party, our packages
important is that we keep the same order in all the files